compiler/compile/OMRCompilation.cpp

```diff
@@ -848,6 +848,12 @@ OMR::Compilation::requiresAnalysisOSRPoint(TR::Node *node)
    }
 
 bool
+OMR::Compilation::pendingPushLivenessDuringIlgen()
```

Comment:
This deserves some doxygen to explain when it should be true and what implementing languages need to support before they can set it to true etc

Reply:
A comment has been added.